pkg/sql/catalog/catformat/index.go

```diff
@@ -174,6 +174,10 @@ func indexForDisplay(
 		}
 	}
 
+	if index.Invisible {
+		f.WriteString(" INVISIBLE")
```

Comment:
Naming consistency: consider replacing the `INVISIBLE` SQL keyword with `NOT VISIBLE`.